watertap/examples/flowsheets/case_studies/electroNP/BSM2_electroNP_no_bioP.py

```diff
@@ -0,0 +1,666 @@
+                iscale.set_scaling_factor(var, 1e0)
+
+    for unit in ("R1", "R2", "R3", "R4", "R5", "R6", "R7"):
+        block = getattr(m.fs, unit)
+        iscale.set_scaling_factor(
+            block.control_volume.reactions[0.0].rate_expression, 1e3
+        )
+        iscale.set_scaling_factor(block.cstr_performance_eqn, 1e3)
+        iscale.set_scaling_factor(
+            block.control_volume.rate_reaction_stoichiometry_constraint, 1e3
+        )
+        iscale.set_scaling_factor(block.control_volume.material_balances, 1e3)
+
+    # Apply scaling
+    scale_variables(m)
+    iscale.calculate_scaling_factors(m)
+
+
+def initialize_system(m):
+    # Initialize flowsheet
+    # Apply sequential decomposition - 1 iteration should suffice
+    seq = SequentialDecomposition()
+    seq.options.tear_method = "Direct"
+    seq.options.iterLim = 1
+    seq.options.tear_set = [m.fs.stream5, m.fs.stream10adm]
+
+    G = seq.create_graph(m)
+    # Uncomment this code to see tear set and initialization order
+    order = seq.calculation_order(G)
+    print("Initialization Order")
+    for o in order:
+        print(o[0].name)
+
+    # Initial guesses for flow into first reactor
+    tear_guesses = {
+        "flow_vol": {0: 1.235},
+        "conc_mass_comp": {
+            (0, "S_A"): 0.0007,
+            (0, "S_F"): 0.0004,
+            (0, "S_I"): 0.0575,
+            (0, "S_N2"): 0.05,
+            (0, "S_NH4"): 0.007,
+            (0, "S_NO3"): 0.0035,
+            (0, "S_O2"): 0.00192,
+            (0, "S_PO4"): 0.02,
+            (0, "S_K"): 0.37,
+            (0, "S_Mg"): 0.02,
+            (0, "S_IC"): 0.11,
+            (0, "X_AUT"): 0.12,
+            (0, "X_H"): 3.3,
+            (0, "X_I"): 3.0,
+            (0, "X_PAO"): 2.3,
+            (0, "X_PHA"): 0.06,
+            (0, "X_PP"): 0.75,
+            (0, "X_S"): 0.050,
+        },
+        "temperature": {0: 308.15},
+        "pressure": {0: 101325},
+    }
+
+    tear_guesses2 = {
+        "flow_vol": {0: 0.0027},
+        "conc_mass_comp": {
+            (0, "S_A"): 0.044,
+            (0, "S_F"): 0.15,
+            (0, "S_I"): 0.0575,
+            (0, "S_N2"): 0.035,
+            (0, "S_NH4"): 0.03,
+            (0, "S_NO3"): 0.002,
+            (0, "S_O2"): 0.0012,
+            (0, "S_PO4"): 0.02,
+            (0, "S_K"): 0.38,
+            (0, "S_Mg"): 0.023,
+            (0, "S_IC"): 0.063,
+            (0, "X_AUT"): 0.31,
+            (0, "X_H"): 24.8,
+            (0, "X_I"): 11.8,
+            (0, "X_PAO"): 8.5,
+            (0, "X_PHA"): 0.086,
+            (0, "X_PP"): 2.1,
+            (0, "X_S"): 4.2,
+        },
+        "temperature": {0: 308.15},
+        "pressure": {0: 101325},
+    }
+
+    # Pass the tear_guess to the SD tool
+    seq.set_guesses_for(m.fs.R3.inlet, tear_guesses)
+    seq.set_guesses_for(m.fs.translator_asm2d_adm1.inlet, tear_guesses2)
+
+    def function(unit):
+        unit.initialize(outlvl=idaeslog.INFO, optarg={"bound_push": 1e-2})
```

Comment:
Certainly a good idea if a solve is failing.

Reply:
I've tested this locally and it seems to work!

```suggestion
        unit.initialize(outlvl=idaeslog.INFO, optarg={"bound_push": 1e-2}, solver="ipopt-watertap")
```

Reply:
All tests pass after import `get_solver` from watertap, no need to change the initialization settings. @lbianchi-lbl @MarcusHolly

Reply:
Yes, I believe you're right @luohezhiming. I think I have mixed things up and what resolved the failure was c5e1b6f rather than the `initialize(..., solver="ipopt-watertap")` change.